vale.ini

```
@@ -36,3 +36,14 @@ Canonical.025b-latinisms-to-reconsider = suggest
 Canonical.025c-latinisms-to-avoid = warning
 Canonical.400-Enforce-inclusive-terms = error
 Canonical.500-Repeated-words = NO
+
+[*.rst]
+BasedOnStyles = Canonical
+
+# inline literal roles
+TokenIgnores = :relatedlinks:, ``.+?``, :samp:`.+?`, :file:`.+?`, :command:`.+?`
```

I'd suggest adding the `:doc:` role, too. Its use is not encouraged, but it's valid syntax (and I do have a repo where it's still used because I haven't gotten to replacing it with `:ref:` yet).
```suggestion
TokenIgnores = :relatedlinks:, ``.+?``, :samp:`.+?`, :file:`.+?`, :command:`.+?`, :doc:`.+?`
```